src/components/home/categories.scss

```diff
@@ -65,37 +88,14 @@
   z-index: 2;
 }
 
-// React-slick classes
-.slick-dots {
-  background-color: #f0f0f0;
-  li {
-    button {
-      &::before {
-        font-size: 17px;
-        transform: matrix(0, 1, 1, 0, 0, 0);
-      }
-    }
-  }
-}
-
-.slick-slide.slick-center {
-  .card {
-    transform: scale(1.15);
-  }
-}
-
-.slick-track {
-  margin-bottom: 20px;
-}
-
 @media screen and (max-width: 768) {
```

Comment:
May you use common breakpoints for these media queries?

Reply:
should I do it in a separate branch?

Reply:
If `categories.scss` belongs to different component, you can do it in a seperate branch but I think there will be just a few changes, so I feel like you can do it here as well.